cpp/Doxyfile.in

```diff
@@ -812,7 +813,9 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                =
+# CUML: Ignore these files since they do not participate in the build process
+EXCLUDE                = @CMAKE_CURRENT_SOURCE_DIR@/src/tsne/cannylab/bh.cu \
+                         @CMAKE_CURRENT_SOURCE_DIR@/src_prims/selection/columnWiseSort.cuh
```

Comment:
why exclude this header file?

Reply:
`columnWiseSort.cuh` can be removed from this list but I excluded `src/tsne/cannylab/bh.cu` because it is not used in the build (Not listed in any CMake file). Including it in the build or including it in the docs both generated errors.

Seems like this should either be removed or added to the build before including it in the docs.